spec/endtoend/endtoend_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'open-uri'
+require 'openssl'
+require 'git_helper'
+require 'tsuru_helper'
+
+describe "TsuruEndToEnd" do
+  context "deploying an application" do
+    before(:all) do
+      @tsuru_home = Tempdir.new('tsuru-command')
+      @tsuru_command = TsuruCommandLine.new(
+        {'HOME' => @tsuru_home.path },
+        { :verbose => RSpec.configuration.verbose }
+      )
+
+      @tsuru_api_url = "https://#{RSpec.configuration.target_api_host}"
+      @tsuru_api_url_insecure = "http://#{RSpec.configuration.target_api_host}:8080"
+
+      @tsuru_command.target_add("ci", @tsuru_api_url)
+      @tsuru_command.target_add("ci-insecure", @tsuru_api_url_insecure)
+
+      @tsuru_user = RSpec.configuration.tsuru_user || raise("You must set 'TSURU_USER' env var")
+      @tsuru_pass = RSpec.configuration.tsuru_pass || raise("You must set 'TSURU_PASS' env var")
+
```

Comment:
Error message should be updated here to reflect that we're getting USER/PASS from the vault rather than as environment variables.